.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
```

Comment:
This order is the default git issue template. It's probably what people are most used to when they use issues, considering many repos use the default template.

Reply:
Still it should make reporting easier with this change, since you dont have to explain what you expected and what happened in "Describe the bug".

Reply:
I feel like the flow is fine?

They encounter a bug.
-> They file a bug report
-> They first explain what the bug is
-> Then how it happened
-> Then what they actually expected instead of this bug

Reply:
I would prefer
-> They first explain what the bug is
-> Explain the step that they has done before the bug happened.
-> What they expected it to do
-> What actually happened

Reply:
What you're asking is the exact same as what the template already does, no? The "What actually happened" is the only difference, and that's implicit in the bug explanation.

Reply:
So let make the order become and probably rename it to:
-> Description
-> Reproduction
-> Expected behavior
-> Actual behavior
-> Other/Extra (or Additional Context)

How would you feel about this?

Reply:
The current flow is:

-> Description
-> Reproduction
-> Expected behavior
-> Screenshots
-> Version Info
-> Additional Context

Your suggestion adds the following:

-> Description
-> Reproduction
-> Expected behavior
-> Actual behaviour <----
-> Screenshots
-> Version Info
-> Additional Context

However, I feel like description already does this, no? I'm a bit confused by the difference. Could you explain how you see it?

Reply:
I would love for it to be like this cheat engine GitHub issue (just the template): https://github.com/cheat-engine/cheat-engine/issues/2582

Reply:
Fair. With that example I can somewhat see how it might be useful. I'll add it in. Thanks for showcasing an example!